pkg/operator/k8sutil/network.go

```diff
@@ -34,7 +41,7 @@ type NetworkAttachmentConfig struct {
 	Mode       string `json:"mode"`
 	Ipam       struct {
 		Type       string `json:"type"`
-		Subnet     string `json:"subnet"`
+		Range      string `json:"range"`
```

Comment:
Make this a separate commit.

Reply:
I think this should be in a separate PR and should be rebased to the releases that have multus support

Reply:
https://github.com/rook/rook/pull/5864